src/app/pages/jails/jail-list/jail-list.component.ts

```diff
@@ -303,21 +309,13 @@ export class JailListComponent implements OnInit {
         id: "update",
         label: T("Update"),
         onClick: (row) => {
-          this.loader.open();
-          this.entityList.busy =
-            this.ws.job('jail.update_to_latest_patch', [row.host_hostuuid]).subscribe(
-              (res) => {
-                this.loader.close();
-                if (res.state == 'SUCCESS') {
-                  this.snackBar.open(T("Jail updated successfully."), T("Close"), { duration: 5000 });
-                } else {
-                  new EntityUtils().handleWSError(this.entityList, res, this.dialogService);
-                }
-              },
-              (res) => {
-                this.loader.close();
-                new EntityUtils().handleWSError(this.entityList, res, this.dialogService);
-              });
+          const dialogRef = this.dialog.open(EntityJobComponent, { data: { "title": T("Updating Jail") }, disableClose: true });
+          dialogRef.componentInstance.setCall('jail.update_to_latest_patch', [row.host_hostuuid]);
+          dialogRef.componentInstance.submit();
+          dialogRef.componentInstance.success.subscribe((res) => {
+            dialogRef.close(true);
+            this.snackBar.open(T("Successfully updated jail."), T('Close'), { duration: 5000 });
```

Comment:
It would be useful to mention the name of the jail that was updated. Also, avoid exclamation points unless something really unexpected happened.  It should not be a surprise that an update succeeded. :)
"Jail jailname successfully updated."

Reply:
Will do it for single jail/plugin update.